substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -700,6 +711,36 @@ def _collector(x):
         if actual_output != expected_output:
             raise Exception('Unexpected output: ' + str(actual_output) + "  !=  " + str(expected_output))
 
+def _debuginfotest(native_image, javac_command, path, build_only, args):
+    mkpath(path)
+    parent = os.path.dirname(path)
+    print("parent=%s"%parent)
+    sourcepath = join(parent, 'src/com.oracle.svm.test/src/')
```

Comment:
No need to hardcode `src/com.oracle.svm.test/src/`. You can use `mx.project('com.oracle.svm.test').source_dirs()`

Reply:
done